src/data/featured.json

```diff
@@ -1,7 +1,7 @@
 {
-	"title": "Samsung ARTIK 710 Comes to resin&#173;.io",
-	"link": "https://resin.io/blog/samsung-artik-710/",
-	"image": "https://resin.io/blog/content/images/2016/10/ARTIK_710_blog.jpg",
-  "text": "The ARTIK 710 is also the first board on resin&#173;.io that is available with the latest resinOS 2.0",
-	"button": "Learn more"
+	"title": "Keeping Your System Running with a Host OS Watchdog",
+	"text": "We are adding a watchdog feature, which helps keep your devices in a healthy state!",
+	"link": "https://resin.io/blog/keeping-your-system-running-watchdog/",
+	"image": "https://resin.io/blog/content/images/2016/12/Download_1_24.png",
```

Comment:
I think it's a good feature article, the image is a tough question, it's not a very visual topic. Some ideas on the watchdog thought (crop them as nice):

- [Pompeii antique watchdog](https://upload.wikimedia.org/wikipedia/commons/thumb/f/fd/Pompeii_-_Casa_di_Paquius_Proculus_-_Watchdog.jpg/898px-Pompeii_-_Casa_di_Paquius_Proculus_-_Watchdog.jpg)
- [Snow and doggie](https://c4.staticflickr.com/8/7294/16542677171_9a2fd17975_b.jpg)
- [going all in with cute](https://pixabay.com/p-316459/?no_redirect)
- [getting tough](https://pixabay.com/p-705991/?no_redirect)

(all above should be free to use, from google image search, free to use with modifications, `watchdog`)

Or something completely different?